src/pages/Login.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import "../App.scss";
+import smilies from "../assets/smilies.png";
+import i6 from "../assets/i6.png";
+import social from "../assets/social.png";
+import play from "../assets/play.png";
+import appStoreButton from "../assets/appstore_button.png";
+import menuIcon from "../assets/icons/menu_icon.png";
+
+function Login() {
+    return (
+      <div class="main-div">
+  
+          <div class="left-container">
+  
+            <div class="left-container__menu">
+              <img src={menuIcon} alt="menu_button"></img>
+            </div>
+  
+            <div class="left-container__smilies">
+              <img src={smilies} alt="smilies"></img>
+            </div>
+  
+            <p class="left-container__app-title">TARGET MVD</p>
+            <p class="left-container__app-subtitle">Find people near you & connect</p>
+            <p class="left-container__description-text" >Create a  target wherever on the map, specify your interest: 
+              Travel, Dating, Music, etc and start conecting with others who share your interest.</p>
+            
+            <form align='center' class="left-container__login-form">
```

Comment:
We usually import the form from its own component

Reply:
Ok I'll do it in the next PR when I implement the login functionality